tools/htseq.wdl

```diff
@@ -29,6 +29,7 @@ task count {
                                          # in this case
 
     Float bam_size = size(bam, "GiB")
+    Float mem_size = bam_size + memory_gb
```

Comment:
You should add an entry for `memory_gb` to the parameter meta to make it clear it's being added to the bam size, instead of an absolute value. Maybe rename the parameter to something like `added_memory_gb` to make that abundantly clear. Not sure about the renaming, but something to consider.